### DISCUSSION TITLE
singleplayer character editing broke

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#13697] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

trying to edit your character when making a new save is no longer possible, more specifically trying to change what face your character has, or changing gendger

### Reproduction steps

try to make a new save and try customizing your characters

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in the latest unstable build, closing
